src/main/java/com/sintern/service/OpenInternPositionServiceImpl.java

```diff
@@ -24,6 +24,17 @@ public OpenInternPositionServiceImpl(OpenInternPositionRepository openInternPosi
         this.openInternPositionRepository = openInternPositionRepository;
     }
 
+    @Override
+    public void addOpenInternPosition(int availablePositions, String department, String description, String name, Company company) {
+        OpenInternPosition openInternPosition = new OpenInternPosition();
```

Comment:
Please delegate this to a `Transformer` (Request -> Model Object, i.e. `OpenInternPosition`) and pass only that object to this method.

Reply:
Done